python3/image.bzl

```diff
@@ -74,6 +74,8 @@ def py3_image(name, base = None, deps = [], layers = [], **kwargs):
     # TODO(mattmoor): Consider using par_binary instead, so that
     # a single target can be used for all three.
 
+    args = kwargs.pop("args", None)
```

Comment:
why is this necessary for py3 image but not for others?

Reply:
Sorry this is still a leftover, I also tried to fix the issue that args are being passed in twice.